### PR TITLE
DAC6-1314-Subtask-DAC6-1344: CheckYourAnswers View pages for Ind and Org

### PR DESCRIPTION
**Pull request by ers2015112**

### CODE REVIEW

File: app/controllers/ChangeIndividualContactDetailsController.scala [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  frontendAppConfig: FrontendAppConfig,
+  identify: IdentifierAction,
+  getData: DataRetrievalAction,
+  requireData: DataRequiredAction,
+  val controllerComponents: MessagesControllerComponents,
+  view: ChangeIndividualContactDetailsView
+) extends FrontendBaseController
+    with I18nSupport {
+
+  def onPageLoad: Action[AnyContent] = (identify andThen getData() andThen requireData) { //ToDo require data when available
```
Comment (matthewharris1) [inline app/controllers/ChangeIndividualContactDetailsController.scala:41]: Remove todo

File: app/controllers/ChangeOrganisationContactDetailsController.scala [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  frontendAppConfig: FrontendAppConfig,
+  identify: IdentifierAction,
+  getData: DataRetrievalAction,
+  requireData: DataRequiredAction,
+  val controllerComponents: MessagesControllerComponents,
+  view: ChangeOrganisationContactDetailsView
+) extends FrontendBaseController
+    with I18nSupport {
+
+  def onPageLoad: Action[AnyContent] = (identify andThen getData.apply andThen requireData) { //ToDo require data when available
```
Comment (matthewharris1) [inline app/controllers/ChangeOrganisationContactDetailsController.scala:41]: Remove todo

File: app/controllers/IndexController.scala [hunk 3]
```diff
@@ -16,22 +16,46 @@
...
+        case Individual   => controllers.routes.ChangeIndividualContactDetailsController.onPageLoad().url
+        case Organisation => controllers.routes.ChangeOrganisationContactDetailsController.onPageLoad().url
+      }
+      {
+        for {
+          userAnswers <- EitherT(subscriptionService.getContactDetails(UserAnswers(request.userId)))
+          _           <- EitherT.right[Throwable](sessionRepository.set(userAnswers))
+        } yield Ok(view(request.subscriptionId, changeDetailsUrl))
+      }.valueOr {
+        _ => Redirect(routes.ThereIsAProblemController.onPageLoad()) //TODO IS THIS RIGHT
```
Comment (matthewharris1) [inline app/controllers/IndexController.scala:58]: is this right? If so remove to do

File: app/views/IndexView.scala.html [hunk 4]
```diff
@@ -34,7 +34,13 @@ <h1 class="govuk-heading-l">@messages("index.heading")</h1>
 
     <p class="govuk-body">@messages("index.p3")</p>
 
-    <p class="govuk-body">@Html(messages("index.p4", link("#", "index.p4.link", openInNewWindow = true)))</p> @*Todo: update link href*@
+    <ul class="govuk-list govuk-list--bullet">
+
+        <li>@Html(messages("index.p4", link("#", "index.p4.link", openInNewWindow = true)))</li>@*Todo: update link href*@
```
Comment (matthewharris1) [inline app/views/IndexView.scala.html:39]: Do we have the link for this yet?

**Review by matthewharris1 — APPROVED** [review]
Just some TODOs still lying about but other than that looks good to me 👍

File: app/controllers/IndexController.scala (hunk 3 above)
Comment (shaileshp0110) [reply app/controllers/IndexController.scala:58]: we don't know at this moment

**Pull request merged**